src/main/java/uk/gov/hmcts/reform/sendletter/info/LetterStatusInfoContributor.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.sendletter.info;
+
+import org.springframework.boot.actuate.info.Info;
+import org.springframework.boot.actuate.info.InfoContributor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.entity.LetterStatus;
+
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toMap;
+
+@Component
+public class LetterStatusInfoContributor implements InfoContributor {
+
+    private final LetterRepository repo;
+
+    public LetterStatusInfoContributor(LetterRepository repo) {
+        this.repo = repo;
+    }
+
+    @Override
+    public void contribute(Info.Builder builder) {
+        builder.withDetail(
+            "letters_by_status",
+            Stream.of(LetterStatus.values())
+                .collect(
+                    toMap(
+                        s -> s.name().toLowerCase(),
+                        s -> repo.countByStatus(s)
```

Comment:
I've started with single query (`group by...`) but it required a custom native query and a new entity model, so I went with this istead.